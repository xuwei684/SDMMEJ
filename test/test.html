<HTML>
<BODY BGCOLOR="#FFEEE0">
<PRE>
0001    chr14_82862432                                    HG_Right_apparent_blunt_join_Snap-back            CTTCTCGAGGATTGGGGACCCTGCGCTGAACATGGAGAACATCACATCAGGATTCCTAGGACCCCTTCTCGTGTTACAGGCGGGGTTTTTCTTGTTG<span style="text-decoration:underline">ACA-TTA</span>CAGAAGGCTGATAACAACTCTA<span style="text-decoration:underline">TAATGT</span>TTTAGTTTTGAATAACATTAAGAGGGCACATATTGATAAGCCACCAAGCGGTTACTGAAGTACTGAAAG
0001    chr14_82862432                                    HG_Right_apparent_blunt_join_Loop-out             CTTCTCGAGGATTGGGGACCCTGCGCTGAACATGGAGAACATCACATCAGGATTCCTAGGACCCCTTCTCGTGTTACAGGCGGGGTTTTTCTTGTTG<span style="text-decoration:underline">ACA-TTA</span>CAGAAGGCTGATAACAACTCTATAATGTTTTAGTTTTGAATA<span style="text-decoration:underline">ACATTA</span>AGAGGGCACATATTGATAAGCCACCAAGCGGTTACTGAAGTACTGAAAG
0002    chr11_91670821                                    HPV_Left_junctional_mh_Loop-out                   CGACCGACCTTG<span style="text-decoration:underline">AGGCATA</span>CTTCAAAGACTGTTTGTTTAAAGACTGGGAGGAGTTGGGGGAGGAGATTAGATTAAAGGTCTTTGTACTAGGAGGCTGT<span style="text-decoration:underline">AG.GCATA</span>A.GTGTTCATCAGGATATTGGCCTAAAGTTTTCTGTTTTGGCTGTCTTTCTGCAAGGTTTTGGTATCAGGATGATGCTTGTCTCATAGAGTGAGTTAGGAAG
0003    chr10_1795623                                     HPV_Left_junctional_mh_Snap-back                  AATTCTTTATAAGGGTCGATGTCCATGCCCCAAAGC<span style="text-decoration:underline">CACC</span>CAAGGCACAGCTTGGAGGCTTGAACAGTAGGACATGAACAAGAGATGATTAGGCAGA<span style="text-decoration:underline">GGT.G</span>AAACAGT.ATCTTTGGAGGAGAAGCACGGAGAACCCATGGAAAGACATCATTTTCCTCATATTGCCCTGGTGTGGCCTTGCAGAGGGATCCTGGCTGAGCAATTCCAA
0003    chr10_1795623                                     HG_Right_junctional_mh_Loop-out                   AATTCTTTATAAGGGTCGATGTCCATGCCCCAAAGCCACCCAAGGCACAGCTTGGAGGCTTGAACAGTAGGACATGAACAAGAGATGATTAGGCAGA<span style="text-decoration:underline">GGT.G</span>AAACAGT.ATCTTTGGAGGAGAAGCACGGAGAACCCATGGAAAGACATCATTTTCCTCATATTGCCC<span style="text-decoration:underline">TGGT</span>GTGGCCTTGCAGAGGGATCCTGGCTGAGCAATTCCAA
0003    chr10_1795623                                     HPV_Left_junctional_mh_Loop-out                   AATTCTTTATAAGGGTCGATGTCCATGCCCCAAAGCCACCCAAGGCACAGCTTGGAGGCT<span style="text-decoration:underline">TGAA</span>CAGTAGGACATGAACAAGAGATGATTAGGCAGAGG<span style="text-decoration:underline">T.GAA</span>ACAGT.ATCTTTGGAGGAGAAGCACGGAGAACCCATGGAAAGACATCATTTTCCTCATATTGCCCTGGTGTGGCCTTGCAGAGGGATCCTGGCTGAGCAATTCCAA
0003    chr10_1795623                                     HPV_Left_junctional_mh_Loop-out                   AATTCTTTATAAGGGTCGATGTCCATGCCCCAAAGCCACCCAAGGCACAGCTTGGAGGCTTGAACAGTAGGACA<span style="text-decoration:underline">TGAA</span>CAAGAGATGATTAGGCAGAGG<span style="text-decoration:underline">T.GAA</span>ACAGT.ATCTTTGGAGGAGAAGCACGGAGAACCCATGGAAAGACATCATTTTCCTCATATTGCCCTGGTGTGGCCTTGCAGAGGGATCCTGGCTGAGCAATTCCAA
0004    chr1_68474310                                     NA                                                TACTTCAAAGACTGTTTGTTTAAAGACTGGGAGGAGTTGGGGGAGGAGATTAGATTAAAGGTCTTTGTACTAGGAGGCTGTAGGCATAAATTGGTCTGCG.CAC.ATTTTACAGATGAGAAGATTCAGAGTTTAAGCTTCCCAAGGACAGAAAGGTAAGTCAGTGGGAAATTCCATGTACATTCCATTACTAAATGCCACATAAC
0005    chr1_68474291                                     NA                                                ATAAGGGTCGATGTCCATGCCCCAAAGCCACCCAAGGCACAGCTTGGAGGCTTGAACAGTAGGACATGAACAAGAGATGATTAGGCAGAGGTGAAAAAGTctttgtactaggaggctgtaggcataaattggtctgcgcaccagcACTGAGTTGTGATTGTAGTGAAAGAAAGGCAATATATTTAAATCTTGAATTTAGCAAGCCCACGCTCGATTTTTATGTCCTTTCCTCTTGCCTTGTATTG
0006    chr1_72295546                                     HG_Left_apparent_blunt_join_Loop-out              GGCCATCTTCACAATTTACGTATATTGCCACAAAGTTTTTCTGATCGACACAGACAATCATCAGTGGTGACAGAAACATAAAAATAC<span style="text-decoration:underline">ATAGATTG</span><span style="text-decoration:underline">ATAGA-TTG</span>GTGGTCTATAAGCTGGAGGAGTGCGAATCCACACTCCGAAAGACACCAAATACTCTATAACGGTTTCTCTTCCAAAAGTGAGACAAGAAATGTGAAA
0007    chr14_82862386                                    HPV_Left_apparent_blunt_join_Loop-out             TGTGTAAATAGTGTCTAGTTTGGAAGTAATGATTAACTAGATGTTCTGGATAATAAGGTTTAATACCCTTATCCAATGGTAAATA<span style="text-decoration:underline">TTTGGT</span>AACC<span style="text-decoration:underline">TTTGG-T</span>TCAAGATGATTCAATGGAGCCTGCCAGTGTGCCTTATCTTATACTTAGGGAATTATTACAGTGAACATGTATTTTTTTTATTCTTGATTATGACTTTGG
0008    chr20_48109321                                    HPV_Left_junctional_mh_Loop-out                   GGTTATGTCATTGGAAGTTATGGGTCCTTGCCACAAGAACACATCATACAAAAAATCAAAGAATGTT<span style="text-decoration:underline">TTAGAAAACT</span>TCCTATTAACAGGCCTATTGA<span style="text-decoration:underline">TT.AGAAA.ACT</span>ACACCTGAATTTCCTAGAATAAATACACTACTTTAAAAAAATTTTAATAGAAAAAAATGCCCAGATAAAGATACCGATCACAGTATCCACATCTATA
0009    chr1_634821                                       HPV_Right_junctional_mh_Snap-back                 AGGGCACTGGCCCCCAACAGGCATCACCCCGCTAAATCCCCTAGAAGTCCCACTCCTAAACACATCCGTATTACTCGCATCAGGAGTATCAATCACC<span style="text-decoration:underline">TGA.G</span><span style="text-decoration:underline">CTCA</span>CCA.GCACCATGCAACTTTTTCACCTCTGCCTAATCATCTCTTGTTCATGTCCTACTGTTCAAGCCTCCAAGCTGTGCCTTGGGTGGCTTTGGGGCATGGACAT
0009    chr1_634821                                       HPV_Right_junctional_mh_Loop-out                  AGGGCACTGGCCCCCAACAGGCATCACCCCGCTAAATCCCCTAGAAGTCCCACTCCTAAACACATCCGTATTACTCGCATCAGGAGTATCAATCACCTG<span style="text-decoration:underline">A.GCT</span>CACCA.GCACCATGCAACTTTTTCACCTCTGCCTAATCATCTCTTGTTCATGTCCTACTGTTCAAGCCTCC<span style="text-decoration:underline">AAGC</span>TGTGCCTTGGGTGGCTTTGGGGCATGGACAT
0010    chr10_1795085                                     HPV_Left_junctional_mh_Loop-out                   AACACAGCAAATCCAGATTGGGACTTCAATCCCAACAAGGA<span style="text-decoration:underline">CACC</span>TGGCCAGACGCCAACAAGGTAGGAGCTGGAGCATTCGGGCTGGGTTTCACCC<span style="text-decoration:underline">CAC.C</span>G.ATGAATGTGTCACCCTATAGAGAGATGAATGTTGGGCAGTCGTCCTGAAGGTCTGACAGGTGAATATCAAGCAGTATAAACAGGAAGTAACTGGGAGAGA
0010    chr10_1795085                                     HPV_Left_junctional_mh_Loop-out                   AACACAGCAAATCCAGATTGGGACTTCAATCCCAACAAGGACACCTGGCCAGACGCCAACAAGGTAGGAGCTGGAGCATTCGGGCTGGGTTT<span style="text-decoration:underline">CACC</span>C<span style="text-decoration:underline">CAC.C</span>G.ATGAATGTGTCACCCTATAGAGAGATGAATGTTGGGCAGTCGTCCTGAAGGTCTGACAGGTGAATATCAAGCAGTATAAACAGGAAGTAACTGGGAGAGA
0010    chr10_1795085                                     HG_Right_junctional_mh_Loop-out                   AACACAGCAAATCCAGATTGGGACTTCAATCCCAACAAGGACACCTGGCCAGACGCCAACAAGGTAGGAGCTGGAGCATTCGGGCTGGGTTTCACCC<span style="text-decoration:underline">CAC.C</span>G.ATGAATGTG<span style="text-decoration:underline">TCAC</span>CCTATAGAGAGATGAATGTTGGGCAGTCGTCCTGAAGGTCTGACAGGTGAATATCAAGCAGTATAAACAGGAAGTAACTGGGAGAGA
0010    chr10_1795085                                     HG_Right_junctional_mh_Snap-back                  AACACAGCAAATCCAGATTGGGACTTCAATCCCAACAAGGACACCTGGCCAGACGCCAACAAGGTAGGAGCTGGAGCATTCGGGCTGGGTTTCACCC<span style="text-decoration:underline">CAC.C</span>G.ATGAATGTGTCACCCTATAGAGAGATGAATGTTGGGCAGTCGTCCTGAAGGTCTGAC<span style="text-decoration:underline">AGGT</span>GAATATCAAGCAGTATAAACAGGAAGTAACTGGGAGAGA
0010    chr10_1795085                                     HPV_Left_junctional_mh_Snap-back                  AACACAGCAAATCCAGATTGGGACTTCAATCCCAACAAGGACACCTGGCCAGACGCCAACAAGGTAGGAGCTGGAGCAT<span style="text-decoration:underline">TCGG</span>GCTGGGTTTCACCCCA<span style="text-decoration:underline">C.CG.A</span>TGAATGTGTCACCCTATAGAGAGATGAATGTTGGGCAGTCGTCCTGAAGGTCTGACAGGTGAATATCAAGCAGTATAAACAGGAAGTAACTGGGAGAGA
0011    chr20_48046784                                    HG_Right_junctional_mh_Snap-back                  CTCTATAACGGTTTCTCTTCTAAAAGTGAGACAAGAAATGTGAAACCACAAGAGTTGCCTGAACTTTAGGCCCATATTAGTGTTGACATAACTGACTA<span style="text-decoration:underline">CT.AGG.A</span>AGGTCACCCTCCTGAATGTG<span style="text-decoration:underline">TCCTAG</span>AATATCTATAGTTTTTCATAAGACTGAGAAAAGCTTTGGATTGCCACCAGATTACTTTGCGGACAACTCTTGC
0012    chrX_111009034                                    HG_Right_junctional_mh_Snap-back                  AAAGACTGGGAGGAGTTGGGGGAGGAGATTAGATTAAAGGTCTTTGTACTAGGAGGCTGTAGGCATAAATTGGTCTGCGCACCAGCACCATGCAACT<span style="text-decoration:underline">TTT.T</span>.TGAAG<span style="text-decoration:underline">TAAA</span>ATTCATATAACAGAAAATTTATCATTTTAACATGAAGAATTCAGTGACATTTGGTACATTCACAGTGTTACAGAACCACAGCTCTTCTGAA
0012    chrX_111009034                                    HG_Right_junctional_mh_Snap-back                  AAAGACTGGGAGGAGTTGGGGGAGGAGATTAGATTAAAGGTCTTTGTACTAGGAGGCTGTAGGCATAAATTGGTCTGCGCACCAGCACCATGCAACT<span style="text-decoration:underline">TTT.T</span>.TGAAGTAAAATTCATATAACA<span style="text-decoration:underline">GAAA</span>ATTTATCATTTTAACATGAAGAATTCAGTGACATTTGGTACATTCACAGTGTTACAGAACCACAGCTCTTCTGAA
0012    chrX_111009034                                    HG_Right_junctional_mh_Loop-out                   AAAGACTGGGAGGAGTTGGGGGAGGAGATTAGATTAAAGGTCTTTGTACTAGGAGGCTGTAGGCATAAATTGGTCTGCGCACCAGCACCATGCAACT<span style="text-decoration:underline">TTT.T</span>.TGAAGTAAAATTCATATAACAGAAAATTTATC<span style="text-decoration:underline">ATTT</span>TAACATGAAGAATTCAGTGACATTTGGTACATTCACAGTGTTACAGAACCACAGCTCTTCTGAA
0012    chrX_111009034                                    HPV_Left_junctional_mh_Loop-out                   AAAGACTGGGAGGAGTTGGGGGAGGAGATTAGATTAAAGGTC<span style="text-decoration:underline">TTTG</span>TACTAGGAGGCTGTAGGCATAAATTGGTCTGCGCACCAGCACCATGCAACTTT<span style="text-decoration:underline">T.T.TG</span>AAGTAAAATTCATATAACAGAAAATTTATCATTTTAACATGAAGAATTCAGTGACATTTGGTACATTCACAGTGTTACAGAACCACAGCTCTTCTGAA
0012    chrX_111009034                                    HG_Right_junctional_mh_Loop-out                   AAAGACTGGGAGGAGTTGGGGGAGGAGATTAGATTAAAGGTCTTTGTACTAGGAGGCTGTAGGCATAAATTGGTCTGCGCACCAGCACCATGCAACTTT<span style="text-decoration:underline">T.T.TG</span>AAGTAAAATTCATATAACAGAAAATTTATCATTTTAACATGAAGAATTCAGTGACA<span style="text-decoration:underline">TTTG</span>GTACATTCACAGTGTTACAGAACCACAGCTCTTCTGAA
</SPAN></PRE>
</BODY>
</HTML>
