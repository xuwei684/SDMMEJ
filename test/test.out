chr14_82862432	100	0	HPV_HG	CTTCTCGAGGATTGGGGACCCTGCGCTGAACATGGAGAACATCACATCAGGATTCCTAGGACCCCTTCTCGTGTTACAGGCGGGGTTTTTCTTGTTGACA-TTACAGAAGGCTGATAACAACTCTATAATGTTTTAGTTTTGAATAACATTAAGAGGGCACATATTGATAAGCCACCAAGCGGTTACTGAAGTACTGAAAG
chr11_91670821	100	-6	HPV_HG	CGACCGACCTTGAGGCATACTTCAAAGACTGTTTGTTTAAAGACTGGGAGGAGTTGGGGGAGGAGATTAGATTAAAGGTCTTTGTACTAGGAGGCTGTAG.GCATAA.GTGTTCATCAGGATATTGGCCTAAAGTTTTCTGTTTTGGCTGTCTTTCTGCAAGGTTTTGGTATCAGGATGATGCTTGTCTCATAGAGTGAGTTAGGAAG
chr10_1795623	100	-8	HPV_HG	AATTCTTTATAAGGGTCGATGTCCATGCCCCAAAGCCACCCAAGGCACAGCTTGGAGGCTTGAACAGTAGGACATGAACAAGAGATGATTAGGCAGAGGT.GAAACAGT.ATCTTTGGAGGAGAAGCACGGAGAACCCATGGAAAGACATCATTTTCCTCATATTGCCCTGGTGTGGCCTTGCAGAGGGATCCTGGCTGAGCAATTCCAA
chr1_68474310	100	-3	HPV_HG	TACTTCAAAGACTGTTTGTTTAAAGACTGGGAGGAGTTGGGGGAGGAGATTAGATTAAAGGTCTTTGTACTAGGAGGCTGTAGGCATAAATTGGTCTGCG.CAC.ATTTTACAGATGAGAAGATTCAGAGTTTAAGCTTCCCAAGGACAGAAAGGTAAGTCAGTGGGAAATTCCATGTACATTCCATTACTAAATGCCACATAAC
chr1_68474291	100	45	HPV_HG	ATAAGGGTCGATGTCCATGCCCCAAAGCCACCCAAGGCACAGCTTGGAGGCTTGAACAGTAGGACATGAACAAGAGATGATTAGGCAGAGGTGAAAAAGTctttgtactaggaggctgtaggcataaattggtctgcgcaccagcACTGAGTTGTGATTGTAGTGAAAGAAAGGCAATATATTTAAATCTTGAATTTAGCAAGCCCACGCTCGATTTTTATGTCCTTTCCTCTTGCCTTGTATTG
chr1_72295546	100	0	HG_HPV	GGCCATCTTCACAATTTACGTATATTGCCACAAAGTTTTTCTGATCGACACAGACAATCATCAGTGGTGACAGAAACATAAAAATACATAGATTGATAGA-TTGGTGGTCTATAAGCTGGAGGAGTGCGAATCCACACTCCGAAAGACACCAAATACTCTATAACGGTTTCTCTTCCAAAAGTGAGACAAGAAATGTGAAA
chr14_82862386	100	0	HPV_HG	TGTGTAAATAGTGTCTAGTTTGGAAGTAATGATTAACTAGATGTTCTGGATAATAAGGTTTAATACCCTTATCCAATGGTAAATATTTGGTAACCTTTGG-TTCAAGATGATTCAATGGAGCCTGCCAGTGTGCCTTATCTTATACTTAGGGAATTATTACAGTGAACATGTATTTTTTTTATTCTTGATTATGACTTTGG
chr20_48109321	100	-5	HPV_HG	GGTTATGTCATTGGAAGTTATGGGTCCTTGCCACAAGAACACATCATACAAAAAATCAAAGAATGTTTTAGAAAACTTCCTATTAACAGGCCTATTGATT.AGAAA.ACTACACCTGAATTTCCTAGAATAAATACACTACTTTAAAAAAATTTTAATAGAAAAAAATGCCCAGATAAAGATACCGATCACAGTATCCACATCTATA
chr1_634821	100	-8	HG_HPV	AGGGCACTGGCCCCCAACAGGCATCACCCCGCTAAATCCCCTAGAAGTCCCACTCCTAAACACATCCGTATTACTCGCATCAGGAGTATCAATCACCTGA.GCTCACCA.GCACCATGCAACTTTTTCACCTCTGCCTAATCATCTCTTGTTCATGTCCTACTGTTCAAGCCTCCAAGCTGTGCCTTGGGTGGCTTTGGGGCATGGACAT
chr10_1795085	100	-2	HPV_HG	AACACAGCAAATCCAGATTGGGACTTCAATCCCAACAAGGACACCTGGCCAGACGCCAACAAGGTAGGAGCTGGAGCATTCGGGCTGGGTTTCACCCCAC.CG.ATGAATGTGTCACCCTATAGAGAGATGAATGTTGGGCAGTCGTCCTGAAGGTCTGACAGGTGAATATCAAGCAGTATAAACAGGAAGTAACTGGGAGAGA
chr20_48046784	100	-3	HPV_HG	CTCTATAACGGTTTCTCTTCTAAAAGTGAGACAAGAAATGTGAAACCACAAGAGTTGCCTGAACTTTAGGCCCATATTAGTGTTGACATAACTGACTACT.AGG.AAGGTCACCCTCCTGAATGTGTCCTAGAATATCTATAGTTTTTCATAAGACTGAGAAAAGCTTTGGATTGCCACCAGATTACTTTGCGGACAACTCTTGC
chrX_111009034	100	-1	HPV_HG	AAAGACTGGGAGGAGTTGGGGGAGGAGATTAGATTAAAGGTCTTTGTACTAGGAGGCTGTAGGCATAAATTGGTCTGCGCACCAGCACCATGCAACTTTT.T.TGAAGTAAAATTCATATAACAGAAAATTTATCATTTTAACATGAAGAATTCAGTGACATTTGGTACATTCACAGTGTTACAGAACCACAGCTCTTCTGAA
